assets/src/components/ladder.tsx

```diff
@@ -175,11 +184,32 @@ const associatedVehicleId = (
   return vehicleOrIncomingGhostVehicleId.replace(ghostIncomingRegex, "")
 }
 
+const VehicleOrCrowdingSvg = ({
```

Comment:
With the most recent round of changes, this has become a very thin wrapper, and could probably be removed.